blueprints/app/files/tests/helpers/start-app.js

```diff
@@ -3,14 +3,20 @@ import Application from '../../app';
 import Router from '../../router';
 import config from '../../config/environment';
 
-export default function startApp(attrs) {
+export default function startApp(attrs, cb) {
   var application;
 
   var attributes = Ember.merge({}, config.APP);
   attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
 
   Ember.run(function() {
-    application = Application.create(attributes);
+    application = Application.extend({
+      init() {
+        this._super(...arguments);
```

Comment:
Although I wrote this. This syntax assumes ember-cli-babel which is optional. So it will need to be restored to es3 safe syntax